mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/course/ChapterContentMenu.java

```diff
@@ -2,12 +2,13 @@
 
 import org.motechproject.mds.annotations.Entity;
 import org.motechproject.mds.annotations.Field;
+import org.motechproject.mds.domain.MdsEntity;
 
 /**
  * Contains the details about the menu file to be played at the end of the chapter
  */
 @Entity
-public class ChapterContentMenu {
+public class ChapterContentMenu extends MdsEntity {
```

Comment:
It gives you access to the auto-gen MDS fields like creationDate, id, etc
On May 7, 2015 8:41 AM, "Frank Huster" <notifications@github.com> wrote:

> In
> mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/course/ChapterContentMenu.java
> <https://github.com/motech-implementations/mim/pull/222#discussion_r29864481>
> :
>
> >
> >  /**
> >   * Contains the details about the menu file to be played at the end of the chapter
> >   */
> >  @Entity
> > -public class ChapterContentMenu {
> > +public class ChapterContentMenu extends MdsEntity {
>
> Interesting, what does extends MdsEntity do?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/222/files#r29864481>.
>